packages/core/scripts/build/copyFiles.ts

```diff
@@ -51,11 +50,7 @@ async function copy(task: CopyTask): Promise<void> {
     const dst = path.resolve(outRoot, task.destination ?? task.target)
 
     try {
-        await fs.copy(src, dst, {
-            recursive: true,
-            overwrite: true,
-            errorOnExist: false,
-        })
+        await fs.copy(src, dst)
```

Comment:
fs.cp https://stackoverflow.com/a/69807672/152142

Or if that isn't possible (e.g. because we need to opt-in to node 16 "experimental" flag) then we can revisit `copyFiles.ts` later.